src/components/molecules/TableComponent/TableComponent.module.scss

```diff
@@ -0,0 +1,80 @@
+@use "scss/attendee/space";
```

Comment:
Undeleted empty file `src/components/molecules/TableComponent/TableComponent.scss`

Reply:
It's already gone: https://github.com/sparkletown/sparkle/tree/7dd0bb3c932237b1f0242e8005218b63f5799cc5/src/components/molecules/TableComponent